Program.cs

```diff
@@ -18,10 +23,9 @@ public static async Task Main()
                 IRepositoryApiClient client;
 
                 // Scope(s) requested by the app
-                string scope = "repository.Read";
                 if (config.AuthorizationType == AuthorizationType.CLOUD_ACCESS_KEY)
                 {
-                    client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey, scope);
+                    client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
```

Comment:
Scope is required after v1. For future compatibility, we should add both repository.read and repository.write scopes instead of removing it?

Reply:
ok added it back for all 3 sample projects